test/testcafe/regression-tests.js

```diff
@@ -1,12 +1,35 @@
 import {ClientFunction, RequestLogger, Selector} from 'testcafe';
 import fs from 'fs';
+import { v4 } from 'uuid';
 
-
+let testSuiteRunId = undefined;
 const getLocation = ClientFunction(() => document.location.href);
 const script_loader_function = new Function(fs.readFileSync('./test/testcafe/test-snippet.js').toString());
+const setProdPerfectCookie = ClientFunction( (id, name, testSuiteRunId, env) => {
+  const data = {
+    test_run_data: {
+      test_run_id: id,
+      test_suite_id: testSuiteRunId,
```

Comment:
this is confusing, is it an identifier for the suite or the run id? and how is it different from the line above

Reply:
sorry, should have been `test_suite_run_id`, indicating the entireity of the run, whereas the line above is for one test in particular

Reply:
excellent - let's do that, and lets' call the line above it test_script_run_id